app/jobs/in_person/send_proofing_notification_job.rb

```diff
@@ -72,12 +74,14 @@ def handle_telephony_response(enrollment:, phone:, telephony_response:)
     end
 
     def notification_message(enrollment:)
-      proof_date = enrollment.proofed_at ? I18n.l(enrollment.proofed_at, format: :sms_date) : 'NA'
-      I18n.t(
-        'telephony.confirmation_ipp_enrollment_result.sms',
-        app_name: APP_NAME,
-        proof_date: proof_date,
-      )
+      with_user_locale(enrollment.user) do
+        proof_date = enrollment.proofed_at ? I18n.l(enrollment.proofed_at, format: :sms_date) : 'NA'
```

Comment:
does `NA` need to be translated? or is it identical in all our locals?

Reply:
If `proof_date` is nullable, I feel like it would probably be easier to have two versions of the string (one with it, one without it) than to try to match translating the NA in context like that

Reply:
yeah, I'm following up on this with the team. I don't think it is nullable, and if it is, we should just send a message without the date, not with "NA" inserted in the date's place.

Reply:
fixed in 93940c30388d0ee376a55f878fda84a3c8f6be3e